src/cascadia/TerminalSettingsModel/KeyChordSerialization.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include "KeyChordSerialization.g.h"
+
+namespace winrt::Microsoft::Terminal::Settings::Model::implementation
+{
+    struct KeyChordSerialization
+    {
+        KeyChordSerialization() = default;

```

Comment:
does this need to be constructable at all?

Reply:
Yeah. Unfortunately static runtimeclasses still need a ctor. Minorly annoyed by that haha

Reply:
I don't recall this conversation coming to a close. *Did* you try it without both the factory _and_ the default ctor? There _must_ be a way to do this.,

Reply:
(?)

Reply:
(Oh huh, sorry I missed this one.)

I tried it and that didn't let me do it. The g.cpp file is looking for the `factory_implementation` so that it can `winrt::make` it. See below:

```c++
// KeyChordSerialization.g.cpp
void* winrt_make_Microsoft_Terminal_Settings_Model_KeyChordSerialization()
{
    return winrt::detach_abi(winrt::make<winrt::Microsoft::Terminal::Settings::Model::factory_implementation::KeyChordSerialization>());
}
```

Here's the errors from the output log:
```
14>KeyMappingSerialization.cpp
14>D:\Terminal\src\cascadia\TerminalSettingsModel\Generated Files\KeyChordSerialization.g.cpp(5,111): error C2039: 'KeyChordSerialization': is not a member of 'winrt::Microsoft::Terminal::Settings::Model::factory_implementation' (compiling source file KeyChordSerialization.cpp)
14>D:\Terminal\src\cascadia\TerminalSettingsModel\KeyChordSerialization.h(20): message : see declaration of 'winrt::Microsoft::Terminal::Settings::Model::factory_implementation' (compiling source file KeyChordSerialization.cpp)
14>D:\Terminal\src\cascadia\TerminalSettingsModel\Generated Files\KeyChordSerialization.g.cpp(5,132): error C2065: 'KeyChordSerialization': undeclared identifier (compiling source file KeyChordSerialization.cpp)
14>D:\Terminal\src\cascadia\TerminalSettingsModel\Generated Files\KeyChordSerialization.g.cpp(5,37): error C2672: 'winrt::make': no matching overloaded function found (compiling source file KeyChordSerialization.cpp)
14>D:\Terminal\src\cascadia\TerminalSettingsModel\Generated Files\KeyChordSerialization.g.cpp(5,134): error C2974: 'winrt::make': invalid template argument for 'D', type expected (compiling source file KeyChordSerialization.cpp)
14>D:\Terminal\src\cascadia\TerminalSettingsModel\Generated Files\winrt\base.h(7401): message : see declaration of 'winrt::make' (compiling source file KeyChordSerialization.cpp)
14>PowershellCoreProfileGenerator.cpp
```